mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -480,10 +640,14 @@ object LinearRegression extends DefaultParamsReadable[LinearRegression] {
 class LinearRegressionModel private[ml] (
     @Since("1.4.0") override val uid: String,
     @Since("2.0.0") val coefficients: Vector,
-    @Since("1.3.0") val intercept: Double)
+    @Since("1.3.0") val intercept: Double,
+    @Since("2.3.0") val scale: Double)
   extends RegressionModel[Vector, LinearRegressionModel]
   with LinearRegressionParams with MLWritable {
 
+  def this(uid: String, coefficients: Vector, intercept: Double) =
+    this(uid, coefficients, intercept, 1.0)
```

Comment:
is it better to set default scale to a impossible value like 0, or -1

Reply:
```scale``` denotes that ```|y - X'w - c|``` is scaled down, I think it make sense to be set 1.0 for least squares regression.